doc/source/cluster/running-applications/job-submission/quickstart.rst

```diff
@@ -308,6 +308,6 @@ Now let's try it with a runtime environment that pins the version of the ``reque
     If both the Driver and Job specify a runtime environment, Ray tries to merge them and raises an exception if they conflict.
     See :ref:`runtime environments <runtime-environments-job-conflict>` for more details.
 
-- The full API reference for the Ray Jobs CLI can be found :ref:`here <ray-job-submission-cli-ref>`. 
-- The full API reference for the Ray Jobs SDK can be found :ref:`here <ray-job-submission-sdk-ref>`.
-- For more information, check out the guides for :ref:`programmatic job submission <ray-job-sdk>` and :ref:`job submission using REST <ray-job-rest-api>`.
+- See :ref:`Ray Jobs CLI <ray-job-submission-cli-ref>` for a full API reference of the CLI. 
+- See :ref:`Ray Jobs SDK <ray-job-submission-sdk-ref>`for a full API reference of the SDK.
```

Comment:
```suggestion
- See :ref:`Ray Jobs SDK <ray-job-submission-sdk-ref>` for a full API reference of the SDK.
```

Reply:
Not sure if this is related ot the doc build failure, not sure what could be causing it